### DISCUSSION TITLE
Switching from Homebrew to the App Store

### DISCUSSION THREAD

**Discussion by apastuszak** [glushchenko/fsnotes#1549] [Q&A] [answered]
Ok, this is a bit of an odd situation here.

I installed fsnotes from homebrew, and decided to switch to it about 15 min later.

To support the project, I uninstalled it using `brew remove fsnotes` and then bought it in the app store.

I then began porting all my notes out of Apple Notes and Obsidian into fsnotes.

This morning I ran my usual brew update with `brew update && brew upgrade` and Homebrew wanted to update my copy of fsnotes.

So now I think I want to remove fsnotes from the App Store, reinstall it using homebrew and then remove it again and reinstall it from the app store.

My question is, will all my settings survive this process? I don't know if removing an App Store app cleans up all the settings and other folders inside of `~/Library`.

**Comment by gingerbeardman** [comment, marked as answer]
`brew remove fsnotes` (or `brew uninstall fsnotes`) will both uninstall the Cask/receipt and remove applications which were moved to /Applications.

So I'm not sure what has happened with your process.

But none of these installs and uninstalls will affect your notes. (of course, you do have a backup, right?)

Settings/preferences may need to be reset, it depends.

**Reply by apastuszak** [reply]
That worked!  Thank you!
